ui/test/nuxeo-error.test.js

```diff
@@ -0,0 +1,65 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import '@webcomponents/html-imports/html-imports.min.js';
+import {
+  fixture,
+  flush,
+  html,
+  isElementVisible,
+  waitForAttrMutation,
+} from '@nuxeo/nuxeo-elements/test/test-helpers.js';
+import { Polymer } from '@polymer/polymer/polymer-legacy.js';
+import '../nuxeo-layout.js';
+import '../widgets/nuxeo-input.js';
+/* eslint-disable no-unused-expressions */
+
+// Export Polymer and PolymerElement for 1.x and 2.x compat
+window.Polymer = Polymer;
+
+suite('<nuxeo-error>', () => {
+  test('Should display error messages when layout is not found', async () => {
+    const layout = await fixture(
+      html`
+        <nuxeo-layout href="notfound.html"></nuxeo-layout>
```

Comment:
Since this is a test of `nuxeo-error`, we should avoid using other elements unless the element does not work without them. In this case, we're just using the `nuxeo-layout` to get the `nuxeo-error`, so we can just stamp a `nuxeo-error` and ditch the `nuxeo-layout`. We can set the very same message, code and url.